van-gogh-second-page/maria/index-Van-Gogh-for-Children.html

```diff
@@ -0,0 +1,226 @@
+                    <p class="note"></p>
+                    <h3> Colouring Page Almond Blossom →</h3>
+                   
+            </div>
+            <div class="card">
+                <div class="picture">
+                <img src="./img/card6.jpeg" alt="альтернативный текст">
+                </div>
+                    <p class="note"></p>
+                    <h3>Colouring Page Self-Portrait with Felt Hat →</h3>
+                    
+            </div>
+            
+        </div>
+    </div>
+
+    <div class="container">
+        <h2 id="container-3">Craft & Play</h2>
+        <div class="container-of-cards">
+            <div class="card">
+                <div class="picture">
+                <img src="./img/card7.jpg" alt="альтернативный текст">
+                </div>
+                    <p class="note"></p>
+                    <h3>Colouring Puzzles →</h3>
+                    <p class="card-description">Colour in, cut out and start puzzling with Vincent van Gogh's masterpieces!</p>
+            </div>
+            <div class="card">
+                <div class="picture">
+                <img src="./img/card8.jpg" alt="альтернативный текст">
+                </div>
+                    <p class="note"></p>
+                    <h3>Vincent van Gogh Board Game →</h3>
+                    <p class="card-description">Play the board game together and follow Vincent van Gogh's journey through the places he lived and worked.</p>
+            </div>
+            <div class="card">
+                <div class="picture">
+                <img src="./img/card9.jpg" alt="альтернативный текст">
+                </div>
+                    <p class="note"></p>
+                    <h3>Roulin Family Diorama →</h3>
+                    <p class="card-description"> Create a shoebox diorama from the Roulin family colouring page, featuring the entire family.
+                    </p>
+            </div>
+        </div>
+    </div>
+
+    <div class="container">
+        <h2 id="container-4">StoryZoo</h2>
+        <div class="container-of-cards">
+            <div class="card">
+                <div class="picture">
+                <img src="./img/card10.jpg" alt="альтернативный текст">
+                </div>
+                    <p class="note">YouTube</p>
+                    <h3>StoryZoo Adventures →</h3>
+                    <p class="card-description">Join Bax, Pepper and Toby on their adventures at the Van Gogh Museum. Watch all 5 videos on YouTube.</p>
+            </div>
+            <div class="card">
+                <div class="picture">
+                <img src="./img/card11.jpg" alt="альтернативный текст">
+                </div>
+                    <p class="note">YouTube</p>
+                    <h3>Enjoy the Museum from Home →</h3>
+                    <p class="card-description">Find more Van Gogh inspired activities to do at home, for all ages.</p>
+            </div>
+            <div class="hide">
```

Comment:
Is this card different from the other?

Reply:
I made this because I want to keep empty space and anoter two cards on the left side of box